model/src/main/java/uk/gov/pay/commons/model/directdebit/agreement/AgreementType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.pay.commons.model.directdebit.agreement;
+
+public enum AgreementType {
+    ON_DEMAND,
+    ONE_OFF
```

Comment:
we might want to add the `fromString` method which we have in almost all of enums at some point, but it's ok for now